packages/storefront-events-collector/src/utils/aep/productListItems.ts

```diff
@@ -22,14 +26,14 @@ const createProductListItems = (
                     value: String(option.valueLabel),
                 });
             });
-
+            // debugger;
             const productListItem: ProductListItem = {
                 SKU: item.product?.sku,
                 name: item.product?.name,
                 quantity: item.quantity,
-                priceTotal: (item.prices?.price?.value * item.quantity) || 0,
+                priceTotal: item.prices?.price?.value * item.quantity || 0,
                 productImageUrl: item.product.mainImageUrl,
-                currencyCode: item.prices.price.currency ?? storefrontContext.storeViewCurrencyCode,
+                currencyCode: item.prices?.price.currency ?? storefrontContext.storeViewCurrencyCode,
```

Comment:
Is `item.prices.price` guaranteed to be present? It's nullable on line 34

Reply:
according to the [shopping cart type](https://github.com/adobe/commerce-events/blob/main/packages/storefront-events-sdk/src/types/schemas/shoppingCart.ts#L23), `prices.price` should always be defined, but `prices.value` is not. in fact it's not even defined in the type